android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt

```diff
@@ -224,18 +233,12 @@ class RNGestureHandlerButtonViewManager : ViewGroupManager<ButtonViewGroup>(), R
         foreground = selectable
         if (_backgroundColor != Color.TRANSPARENT) {
           setBackgroundColor(_backgroundColor)
```

Comment:
```suggestion
```

I believe this one is redundant. The intent was probably to call `setBackgroundColor` method from `View`, but the button overrides it, so it ended up doing basically `_backgroundColor = _backgroundColor`.

Changing it to `super.setBackgroundColor()` would solve that, but then we would ignore the border radius logic.

Reply:
I also thought that it won't do anything, but I left it to make sure that the value in that field is correct. But as you said, we can probably ignore that (I don't think it will cause any trouble).

Done in [ebcf766](https://github.com/software-mansion/react-native-gesture-handler/pull/2642/commits/ebcf766d4fbedbfabec8b64f849fa938bc24fb37)